Classes/WPEditorView.m

```diff
@@ -275,8 +275,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [newValue getValue:&newSize];
         
             if (newSize.height != self.lastEditorHeight) {
-                [self refreshVisibleViewportAndContentSize];
+                
+                // First make sure that the content size is not changed without us recalculating it.
+                //
+                self.webView.scrollView.contentSize = CGSizeMake(self.frame.size.width, self.lastEditorHeight);
```

Comment:
It took me a second to realize that we were setting the scrollView's contentSize.height back to the last recorded height.  Maybe I need more coffee?

I'd thought we were trying to use the CG macros for extracting values from a CGRect: `CGRectGetWidth(self.frame)` for instance.  Maybe its not a convention, just more personal style?